typescript/cli/src/deploy/core.ts

```diff
@@ -83,7 +89,15 @@ export async function runCoreDeploy({
     );
   }
   const artifacts = await runArtifactStep(chains, artifactsPath);
-  const multisigConfig = await runIsmStep(chains, ismConfigPath);
+  const result = await runIsmStep(chains, ismConfigPath);
+  // we can either specify the full ISM config or just the multisig config
+  const isAdvancedIsm = isAdvancedISMConfig(result);
+  const ismConfigs = isAdvancedIsm
+    ? (result as ChainMap<IsmConfig>)
+    : undefined;
+  const multisigConfigs = isAdvancedIsm
+    ? defaultMultisigConfigs
+    : (result as ChainMap<MultisigConfig>);
```

Comment:
would prefer if there was only 1 ism config

Reply:
not the same type and multisigConfigs is used by the igp as well